care/facility/api/viewsets/bed.py

```diff
@@ -57,6 +59,29 @@ class BedViewSet(
     search_fields = ["name"]
     filterset_class = BedFilter
 
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        validated_data = serializer.validated_data
+        number_of_beds = validated_data.get("number_of_beds")
+        del validated_data["number_of_beds"]
+        # Bulk creating n number of beds
+        if number_of_beds > 1:
+            objs = []
+            for i in range(1, number_of_beds + 1):
```

Comment:
we should limit number of beds created in bulk
@gigincg

Reply:
@sainak I agree. Let's Limit to 100 at a time?

cc: @siddnikh